package.json

```diff
@@ -9,7 +9,7 @@
     "url": "https://github.com/magento-research/pwa-studio/issues"
   },
   "engines": {
-    "node": ">=8.0.0"
+    "node": ">=8.0.0 <=11.0.0"
```

Comment:
Should this be `<=9.0.0` ?

I know there's still work to be done to get us fully valid on Node 10 - should an update to `<= 11.0.0` wait until that work is complete?

Reply:
Not sure, going by what @zetlen is saying for the project. right now without the addition of the upper limit we're saying that anything above 8 is working, so need some clarity on that.

Reply:
Totally agree, I'm just pointing out that I don't know if we know that 10 is working. Maybe we set our upper limit to 9 until we're sure.

Reply:
I am running 11 and it is working, but that is part of the issue is that right now everyone has different local environments so it is difficult to confirm any of these versions are working as expected. How are we determining a version is cleared as working?